### PR TITLE
Triangle facts

### PR DESCRIPTION
**Pull request by jklemen**

### CODE REVIEW

File: triangle_facts/triangle_facts.rb [hunk 1]
```diff
@@ -1,58 +1,69 @@
...
+  def recite_angles
+    angles = calculate_angles(side1, side2, side3)
+    puts "The angles of this triangle are #{angles.join(',')}"
 
+    puts "This triangle is also a right triangle!" if angles.include? 90
+  end
+
+  def recite_facts
+    recite_type
+    recite_angles
```
Comment (djkotowski) [inline triangle_facts/triangle_facts.rb]: Simple and consistent.  Nice!

File: triangle_facts/triangle_facts.rb [hunk 2]
```diff
@@ -1,58 +1,69 @@
...
+    puts "isosceles!" if isosceles?
+    puts "equilateral!" if equilateral?
+    puts "scalene." if scalene?
+  end
+
+  def recite_angles
+    angles = calculate_angles(side1, side2, side3)
+    puts "The angles of this triangle are #{angles.join(',')}"
 
+    puts "This triangle is also a right triangle!" if angles.include? 90
```
Comment (djkotowski) [inline triangle_facts/triangle_facts.rb]: Wrap args in parentheses.

File: newfile.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+hello, world
```
Comment (djkotowski) [inline newfile.txt]: Make sure your branches are clean.

**Comment by djkotowski** [comment]
Good work!  You're done with this exercise.

File: triangle_facts/triangle_facts.rb (hunk 2 above)
Comment (jklemen) [inline triangle_facts/triangle_facts.rb]: Ah, good catch. Rubocop must think that's okay.

**Pull request closed without merging**